generator/src/main/java/com/scottlogic/deg/generator/outputs/targets/FileOutputTarget.java

```diff
@@ -53,4 +52,8 @@ public FileOutputTarget withFilename(String filename){
             filePath.resolve(dataSetWriter.getFileName(filename)),
             dataSetWriter);
     }
+
+    public Path getFilePath() {
+        return filePath;
```

Comment:
I think your comment addresses this point, but is this method breaking encapsulation a bit.

Maybe - as i think you've elluded to - this should be `boolean fileExists()`?